### PR TITLE
Use CRD datacenters during e2e tests

### PR DESCRIPTION
**Pull request by alvaroaleman**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
none
```

### CODE REVIEW

File: api/cmd/kubermatic-api/main.go [hunk 1]
```diff
@@ -138,14 +138,26 @@ func createInitProviders(options serverRunOptions) (providers, error) {
...
+	if _, err := mgr.GetCache().GetInformer(&kubermaticv1.Seed{}); err != nil {
+		kubermaticlog.Logger.With("error", err).Fatal("failed to get seed informer")
+	}
+	go func() {
+		if err := mgr.Start(wait.NeverStop); err != nil {
+			kubermaticlog.Logger.With("error", err).Fatal("failed to start the mgr")
+		}
+	}()
+	if synced := mgr.GetCache().WaitForCacheSync(wait.NeverStop); !synced {
+		kubermaticlog.Logger.Fatal("failed to sync mgr cache")
```
Comment (mrIncompetent) [inline api/cmd/kubermatic-api/main.go:164]: With `wait.NeverStop` i think it's save to assume this condition will never be hit.
Maybe rather:
`mgr.GetCache().WaitForCacheSync(wait.NeverStop)`
?

**Comment by alvaroaleman** [comment]
/assign @zreigz 

Can you have a look, please?

**Comment by zreigz** [comment]
LGTM
/lgtm
/approve

**Pull request merged**